apps/catalog-admin/components/color-picker/index.tsx

```diff
@@ -31,8 +52,8 @@ export const ColorPicker = ({ defaultColor, type }: ColorPicker) => {
         ></div>
```

Comment:
Empty div?

Reply:
Ja, eller det er en boks med farge 
![image](https://github.com/Informasjonsforvaltning/catalog-frontend/assets/48259307/dc5d652b-eb72-4e4e-a77b-c1bb06100931)

Reply:
OK. Tror da kan du fjerne closing tag 
```suggestion
        />
```